_includes/schedule.html

```diff
@@ -48,7 +48,11 @@ <h5 class="slot-title" itemprop="name">{{ session.title }}</h5>
                                                     {% if session_speaker == speaker.id %}
                                                     <li itemprop="performer"><div class="speaker-img flow-img img-circle" style="background-image: url({{ site.baseurl | append: '/img/people/' | append: speaker.thumbnailUrl }})"></div>
                                                     <p class="speaker-name">{{ speaker.name }} {{ speaker.surname }}
-                                                        <span class="speaker-position">{{ speaker.company }}</span>
+                                                        <span class="speaker-position">
+                                                            {% for company in speaker.company %}
```

Comment:
Can we have this in `p` tags as the text as shown in http://prntscr.com/gb29gk is little bit conjusted